pinot-query-runtime/src/main/java/org/apache/pinot/query/service/server/QueryServer.java

```diff
@@ -71,7 +88,17 @@ public void start() {
     LOGGER.info("Starting QueryServer");
     try {
       if (_server == null) {
-        _server = ServerBuilder.forPort(_port).addService(this).maxInboundMessageSize(MAX_INBOUND_MESSAGE_SIZE).build();
+        if (_tlsConfig == null) {
+          try {
+            _server =
+                ServerBuilder.forPort(_port).addService(this).maxInboundMessageSize(MAX_INBOUND_MESSAGE_SIZE).build();
+          } catch (Exception e) {
+            throw new RuntimeException("Failed to start secure QueryServer", e);
```

Comment:
Seems like a wrong message in the exception. we are trying to start the non-ssl server here, right?

Reply:
Done